src/compat/dbtype.h

```diff
@@ -772,6 +772,7 @@ union db_char
   {
     unsigned char style;
     unsigned char codeset;
+    bool is_max_string;
```

Comment:
@lucianlazar2211 

Please notice that there is `compat/dbi_compat.h` file which is for publishing to users.
Would you also modify it?